tests/library/browsercontext-har.spec.ts

```diff
@@ -358,6 +358,14 @@ it('should record overridden requests to har', async ({ contextFactory, server }
   expect(await page2.evaluate(fetchFunction, { path: '/echo', body: '12' })).toBe('12');
 });
 
+it('should replay requests with multiple set-cookie headers properly', async ({ context, asset }) => {
```

Comment:
Not sure this test is still necessary with the multi-cookie test below (seems like that one both records and replays a HAR)